src/main/java/com/example/TicketBookingApp/exception/WrongSeatException.java

```diff
@@ -0,0 +1,5 @@
+package com.example.TicketBookingApp.exception;
+
+public class WrongSeatException extends RuntimeException{
+    public WrongSeatException() { super("At least 2 space required"); }
```

Comment:
to jedyna regułka dla której ustawienie miejsc może być nieprawidłowe? w sensie message może być bardziej specyficzny w zależności od algorytmu a same wyjątki trzymasz w pakiecie exception który nie pokazuje powiązania w jakich sytuacjach taki exception poleci

Reply:
Rozumiem. Lepszym rozwiązaniem byłoby przesłanie argumentu w konstruktorze wyjątku, w którym przekazana byłaby spersonalizowana regułka.